remote_monitoring/remote_monitoring/metrics_export.py

```diff
@@ -8,11 +8,11 @@
 from typing import Any
 
 from qibocal.auto.output import Output
+from qibocal.calibration.calibration import Calibration
```

Comment:
```suggestion
from qibocal.calibration.calibration import Calibration, CALIBRATION
```
See suggestion later